ffi/ffi_wrapper.go

```diff
@@ -192,6 +197,8 @@ func IsAllowedFor(
 	cResolverAddress *C.char,
 	cToken *C.char,
 ) (*C.char, *C.char) {
+	ctx := context.Background()
+
 	resolverAddress := C.GoString(cResolverAddress)
 	rAdd, err := url.Parse(resolverAddress)
 	if err != nil {
```

Comment:
(staticcheck for below line: no formatting performed - can remove `Sprintf`)

Reply:
nice